acquisitions/src/main/resources/thunderjet/mod-orders/reusable/create-circulation-policy.feature

```diff
@@ -1,18 +1,16 @@
 Feature: Reusable function to initi circulation request
-  # parameters: id, name
 
   Background:
     * url baseUrl
 
   @CreateLoanPolicy
   Scenario: Create loan policy
-
     Given path 'loan-policy-storage/loan-policies'
     And request
       """
       {
-        "id": "#(id)",
-        "name": "#(name)",
+        "id": "d9cd0bed-1b49-4b5e-a7bd-064b8d177231",
```

Comment:
maybe using random uuid is more suitable here?

Reply:
added explanation in the beginning of the file. All @Policy ids is being used in @CirculationRules. If policy id has changes, appopriate id in rules need to be changed.